docs/reference/glossary.md

```diff
@@ -449,8 +468,8 @@ To see example uses of transaction scripts, follow [Move scripts](../move/move-o
 
 ### Version
 
-- A **version** is also called “height” in blockchain literature.
-- A transaction at height 0 is the first transaction (genesis transaction), and a transaction at height 100 is the 101st transaction in the transaction store.
+- A **version** is a sequentially increasing number that increments for every [transaction](#transaction).
```

Comment:
`height` is often what's used for this as well, I think we should probably keep that line in but below the actual explanation.  I've definitely used that to explain it to people.

Reply:
I thought in all of our user facing stuff height refers to block and version refers to transaction

Reply:
Yes, version refers to transactions.

However, to explain what version is, it's similar to block height.  It's an atomically increasing number that gives absolute order.